### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dsalgador**
- Fix some grammar typos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,14 +9,14 @@ For any comments about the tutorial (suggestions for improvement, typos, error r
 Before start working with the tutorial notebook(s), please follow the steps in the [prerequisites & setup section](#prerequisites--setup).
 
 
-After you complete sections 1, 2 and 3, I recommend you doing the proposed Exercises. You can fork this repository and work locally in your forked repo.
+After you complete sections 1, 2 and 3, I recommend you do the proposed Exercises. You can fork this repository and work locally in your forked repo.
 
-The reader is encouraged to provide its solutions to usecases it they are still not present in the `usecases/` directory via a pull request. Read the [usecase solutions guidelines section](#contributing-with-use-cases-requirements-and-guidelines) for more details about how to do it.
+The reader is encouraged to provide its solutions to use cases it they are still not present in the `usecases/` directory via a pull request. Read the [usecase solutions guidelines section](#contributing-with-use-cases-requirements-and-guidelines) for more details about how to do it.
```
Comment (amolerca) [inline README.md:14]: ... its solutions to use cases **if** they are ...

File: README.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ The reader is encouraged to provide its solutions to usecases it they are still
         (blockchain_tutorial_python) jupyter notebook
         ```
 
-        If you where already using jupyter, refreshing one of the notebooks if necessary and going to Kernel --> Change kernel, we should see a "BlockchainTutorial (Conda py3.9)" kernel that we can select.
+        If you were already using jupyter, refreshing one of the notebooks if necessary and going to Kernel --> Change kernel, we should see a "BlockchainTutorial (Conda py3.9)" kernel that we can select.
```
Comment (amolerca) [inline README.md:50]: ... already using **Jupyter** ...

**Review by amolerca — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dsalgador) [reply README.md]: Done f94aa2dd3995377319026e454e99c32bdca00dcc

File: README.md (hunk 2 above)
Comment (dsalgador) [reply README.md]: Done f94aa2dd3995377319026e454e99c32bdca00dcc

**Review by amolerca — APPROVED** [review]
CAC

**Pull request merged**